.gitignore

```diff
@@ -2,4 +2,6 @@
 .htaccess
 .idea/
 sites/*/sqlconf.php
-sites/*/profile_pictures/*
```

Comment:
Do you have all the /sites various document/file directories in the gitignore somewhere else?

Reply:
That wasn't in my pull request -- I just ignore the `backups/` directory.